po/vi.po

```diff
@@ -6911,72 +6911,86 @@ msgstr "\"Dữ liệu đã được ghi: {:?}\""
 
 #: src/std-traits/default.md
 msgid "The `Default` Trait"
-msgstr ""
+msgstr "Trait `Default`"
 
 #: src/std-traits/default.md
 msgid ""
 "[`Default`](https://doc.rust-lang.org/std/default/trait.Default.html) trait "
 "produces a default value for a type."
 msgstr ""
+"Kiểu dữ liệu implement trait [`Default`](https://doc.rust-lang.org/std/"
+"default/trait.Default.html) sẽ sở hữu một giá trị mặc định."
 
 #: src/std-traits/default.md
 msgid "\"John Smith\""
-msgstr ""
+msgstr "\"John Smith\""
 
 #: src/std-traits/default.md
 msgid "\"{default_struct:#?}\""
-msgstr ""
+msgstr "\"{default_struct:#?}\""
 
 #: src/std-traits/default.md
 msgid "\"Y is set!\""
-msgstr ""
+msgstr "\"Biến Y đã được gán!\""
 
 #: src/std-traits/default.md
 msgid "\"{almost_default_struct:#?}\""
-msgstr ""
+msgstr "\"{almost_default_struct:#?}\""
 
 #: src/std-traits/default.md src/lifetimes/exercise.md
 #: src/lifetimes/solution.md
 msgid "\"{:#?}\""
-msgstr ""
+msgstr "\"{:#?}\""
 
 #: src/std-traits/default.md
 msgid ""
 "It can be implemented directly or it can be derived via `#[derive(Default)]`."
 msgstr ""
+"Người dùng có thể trực tiếp implement trait này, hoặc derive trait này bằng "
```

Comment:
I know this trans holds "trait" as a type but since "implement" type isn't here for this one can we use "ứng dụng" or "áp dụng" instead?  Which one do you prefer better and why?

Reply:
In my opinion, when discussing traits, it's better to use the raw term "implement" than the its translated version, such as "ứng dụng" or "áp dụng". This is because using implement implies that the type implementing the trait will have all the necessary method and type signatures. The same doesn't apply for "ứng dụng" or "áp dụng", as we ofter use them in a more generic ways.
I believe the same argument can be used for "inheritance", but unlike "implement trait", Vietnamese literature has used "kế thừa" for a long enough time that Vietnamese programmers understand "kế thừa" to mean "inherit every single method and variable".
I admit that this is not optimal, but until we find a better translation for this word, or the Vietnamese Rust user base grows large enough to accept a canonical translation for it, I think it's best to be explicit about what we mean.

Reply:
Thanks:) I resolved it